core/templates/domain/feedback_message/ThreadMessageObjectFactory.ts

```diff
@@ -88,7 +83,7 @@ export class ThreadMessageObjectFactory {
       threadMessageBackendDict.message_id, threadMessageBackendDict.text,
       threadMessageBackendDict.updated_status,
       threadMessageBackendDict.updated_subject,
-      this.threadMessageSummaryObjectFactory.createNew(
+      ThreadMessageSummary.createNew(
```

Comment:
Can we change "createNew" function to a constructor?

Reply:
@nithusha21 Can you elaborate it please?What do you mean by changing it to a constructor?Also can I refractor this `ThreadMessageObjectFactory` -> `ThreadMessage` in this PR only?

Reply:
@nithusha21 PTAL

Reply:
Sure go ahead and refactor that too... By my comment I meant replace the createNew function with a constructor (as this is the purpose of the constructor). And then you can do 'new ThreadMessageSummary(....)' instead of 'ThreadMessageSummary.createNew(.....)'

Reply:
done